<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderByWidgetForm</class>
 <widget class="QWidget" name="OrderByWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order By</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_29">
     <item row="0" column="0">
      <widget class="QGroupBox" name="m_orderByGroupBox">
       <property name="title">
        <string>Order</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_27">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_26">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_25">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_23">
              <item row="0" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Property</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Order</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QGridLayout" name="gridLayout_24">
              <item row="0" column="0">
               <widget class="QComboBox" name="m_orderPropertyComboBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QComboBox" name="m_orderComboBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <layout class="QGridLayout" name="gridLayout_55">
            <item row="0" column="0">
             <widget class="QPushButton" name="m_addOrderPushButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="m_removeOrderPushButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_15">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_28">
        <item row="0" column="0">
         <widget class="QTableWidget" name="m_orderTableWidget">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>29</number>
          </attribute>
          <column>
           <property name="text">
            <string>Property</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Order</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_orderPropertyComboBox</tabstop>
  <tabstop>m_orderComboBox</tabstop>
  <tabstop>m_addOrderPushButton</tabstop>
  <tabstop>m_removeOrderPushButton</tabstop>
  <tabstop>m_orderTableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
